internal/core/src/segcore/ConcurrentVector.h

```diff
@@ -144,16 +266,26 @@ class ConcurrentVectorImpl : public VectorBase {
     }
 
     void
-    set_data_raw(ssize_t element_offset, const void* source, ssize_t element_count) override {
-        set_data(element_offset, static_cast<const Type*>(source), element_count);
+    fill_chunk_data(const void* source, ssize_t element_count) override {
```

Comment:
this API should be moved to ConcurrentVector.cpp

Reply:
This is the member function of the template class